benchmark_tests/run.py

```diff
@@ -115,28 +110,29 @@ def main():
                         help='Number of layers (for some models only).')
     parser.add_argument('--wide_scale', type=int, default=1,
                         help='Wide scale (for W-ResNet only).')
-    # Swap settings
-    parser.add_argument('--mxnet_engine_type', type=str, default='NaiveEngine',
-                        help='MXNET_ENGINE_TYPE')
-    parser.add_argument('--mxnet_prefetch_algorithm', type=str,
-                        default='ComputePrefetch',
-                        help='Prefetch look ahead steps.')
-    parser.add_argument('--mxnet_mem_mgr_type', type=str, default='CUDA',
-                        help='The memory manager type for the swapping system.')
-    parser.add_argument('--mxnet_prefetch_steps', type=int, default=30,
-                        help='Prefetch look ahead steps.')
-    parser.add_argument('--mxnet_swap_algorithm', type=str,
-                        default='NaiveHistory',
-                        help='Swap algorithm.')
-
+    # Environment variables settings.
+    # NOTE that if b new environment variable is added, it should also be here.
+    parser.add_argument('--envs', type=str, nargs='+',
+                        help='Environment variables.',
+                        default=['PYTHONPATH=/home/fegin ',
+                                 'LD_LIBRARY_PATH=/usr/local/cuda/lib64 ',
+                                 'CUDA_VISIBLE_DEVICES=0 ',
+                                 'MXNET_ENGINE_TYPE=NaiveEngine ',
+                                 'MXNET_PREFETCH_ALGORITHM=ComputePrefetch ',
+                                 'MXNET_PREFETCH_STEPS=30 ',
```

Comment:
The env  var for prefetch steps is actually "MXNET_PREFETCH_STEPS_AHEAD". Now all tests will run with steps=100 (default value in gpu_swap_prefetch.cc). This problem was in the earlier version as well.